src/btree/bt_debug.c

```diff
@@ -942,13 +942,41 @@ __wt_debug_cursor_page(void *cursor_arg, const char *ofile)
     cbt = cursor_arg;
     session = CUR2S(cursor_arg);
 
-    WT_RET(__wt_msg(session, "dumping the current cursor page"));
     WT_WITH_BTREE(session, CUR2BT(cbt), ret = __wt_debug_page(session, NULL, cbt->ref, ofile));
-    if (cbt->last_ref != NULL) {
-        WT_RET(__wt_msg(session, "dumping the last cursor page"));
-        WT_WITH_BTREE(
-          session, CUR2BT(cbt), ret = __wt_debug_page(session, NULL, cbt->last_ref, ofile));
+    return (ret);
+}
+
+/*
+ * __wt_debug_dump_tree --
+ *     Lock all the refs and Dump the in-memory information.
```

Comment:
We don't need to capitalize `Dump`

Reply:
Fixed in 548cc0338987afb463397e5b10e6ebaf1e789f75